docs/usage/exporting_models.md

```diff
@@ -12,25 +12,28 @@ Arguments:
 * `by_alias`: whether field aliases should be used as keys in the returned dictionary; default `False`
 * `exclude_unset`: whether fields which were not explicitly set when creating the model should
   be excluded from the returned dictionary; default `False`.
-  Prior to **v1.0**, `exclude_unset` was known as `skip_defaults`; use of `skip_defaults` is now deprecated
 * `exclude_defaults`: whether fields which are equal to their default values (whether set or otherwise) should
   be excluded from the returned dictionary; default `False`
 * `exclude_none`: whether fields which are equal to `None` should be excluded from the returned dictionary; default
   `False`
+* `round_trip`: whether to enable serialization and deserialization round-trip support; default `False`
+* `warnings`: whether to log warnings when invalid fields are encountered; default `True`
 
 Example:
 
 ```py
-from pydantic import BaseModel
+from typing import Any, List, Optional
+
+from pydantic import BaseModel, Field, Json
 
 
 class BarModel(BaseModel):
     whatever: int
 
 
 class FooBarModel(BaseModel):
-    banana: float
-    foo: str
+    banana: Optional[float] = 1.1
```

Comment:
Why the `Optional` is needed now that you have a default?

Reply:
I've changed it to `Optional` to [add a test for exclude_none](https://github.com/pydantic/pydantic/pull/6030/files#diff-fca1519e75e0a9b8025318f8594ab6a9d512c0dc8116c2d313bbfe92a84949e2R61)